canonical-web-and-design/vanilla-framework
# Add the search and filter component

## Done
Add the search and filter component
- Styling for the component
- A basic example of the pattern
- Documentation for the pattern
- Standalone example

Fixes https://github.com/canonical-web-and-design/vanilla-framework/issues/3144

## QA

- Open /docs/examples/patterns/search-and-filter/default
- Check when focusing on the input it opens the panel and looks correct.
- Review updated documentation:
  - Go to /docs/patterns/search-and-filter and check the content is ok and clear
  - Go to /docs/examples/standalone/patterns/search-and-filter/default and check the stand-alone example renders

## Screenshots
![image](https://user-images.githubusercontent.com/1413534/109867869-ea068d00-7c5e-11eb-95a6-634af1fe2592.png)
